Automattic/wp-calypso
# Hosting Metrics: Add requests by HTTP response code grouped bar chart first pass

See https://github.com/Automattic/dotcom-forge/issues/3356

## Proposed Changes

* It introduces a new uPlot Grouped bar chart to hosting metrics and storybook

## Testing Instructions



* Navigate to `/site-monitoring/${siteSlug}`
* Scroll down
* Observe the new bar chart

**Storybook**

- Run `yarn storybook:start`
- Access http://localhost:6006/?path=/story/packages-components-chart-uplot--bar-chart-uplot
- Observe the Grouped Bar Chart looks like the design

### Screenshots

<img width="1594" alt="Screenshot 2023-08-04 at 17 54 04" src="https://github.com/Automattic/wp-calypso/assets/779993/6b63b0ab-df68-4575-a511-6563cc473bb7">



<img width="1271" alt="Screenshot 2023-08-03 at 19 12 08" src="https://github.com/Automattic/wp-calypso/assets/779993/4f11dfe7-892b-4dee-ba85-535f5f2f31f1">



## Pre-merge Checklist
